opt/migrate_pgsql_db.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
```

Comment:
Probably should add
```suggestion
set -exuo pipefail
```
Definitely need to make sure we do not delete the DB if the export fail. :-)

Reply:
@danielhollas, can you elaborate on what it does? I assume you intend to exit on the first error. However, there are many other letters that I am not familiar with.

Also, on a side note: the backup folder is not deleted by the script.

Reply:
Certainly. `-u` turns undefined variables into errors. This is generally much safer, although sometimes requires workarounds if the variable is indeed undefined. `-x` is the debug mode, the script essentially echoes each command to the terminal (perhaps not useful here). `-o pipefail` makes sure that if any command in a pipe chain fails, the overall pipe command exits with non-zero. Otherwise, the pipe only returns the status of the last command, which might hide errors.

> Also, on a side note: the backup folder is not deleted by the script.
:+1: